components/brave_wallet/browser/brave_wallet_constants.h

```diff
@@ -27,51 +27,93 @@ constexpr int32_t kAutoLockMinutesMin = 1;
 constexpr int32_t kAutoLockMinutesMax = 10080;
 
 // List of assets from Wyre, available to buy
-static base::NoDestructor<std::vector<mojom::BlockchainToken>> kBuyTokens(
+static base::NoDestructor<std::vector<mojom::BlockchainToken>> kWyreBuyTokens(
```

Comment:
Actually these vectors should be refactored to something like
```
std::vector<mojom::BlockchainToken>& GetWyreBuyTokens() {
  static base::NoDestructor<std::vector<mojom::BlockchainToken>> tokens={....}
  return *tokens;
}
```
so they are allocated on first use. Otherwise their initialization slows down browser startup a bit.

Reply:
Fixed. The suggested functions are in the `brave_wallet_constants.cc` file.

fdfd2f5b0d086b4d433080c750993185e3f35f21